src/Exception/EventListener/PimcoreElementNotFoundException.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Valantic\ElasticaBridgeBundle\Exception\EventListener;
+
+use Valantic\ElasticaBridgeBundle\Exception\BaseException;
+
+class PimcoreElementNotFoundException extends BaseException
+{
+    public function __construct(?int $id, string $type)
+    {
+        parent::__construct(sprintf('Pimcore element ID %d of type %s not found', $id, $type), 0, null);
```

Comment:
You don't need to set parent `$code` and `$previous` parameter

Reply:
Copy-pasted from other exceptions, so I fixed it there as well. Thanks!